scripts/cleanup_dp.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -o errexit
+
+mkdir ~/.aws; touch ~/.aws/config
+
+echo "[default]
+aws_access_key_id = $AWS_ACCESS_KEY_ID
+aws_secret_access_key = $AWS_SECRET_ACCESS_KEY" > ~/.aws/credentials
+
+echo "[default]
+[profile eksci]
+role_arn= $AWS_ROLE_ARN
+output = json
+region=ap-south-1
+source_profile = default" > ~/.aws/config
```

Comment:
Same as above, can we not overwrite global config files please?

For `role_arn`, we can provide `--role-arn "$AWS_ROLE_ARN"`. From https://awscli.amazonaws.com/v2/documentation/api/latest/reference/eks/update-kubeconfig.html.

So this can just be:

```suggestion
export AWS_REGION=ap-south-1
export AWS_DEFAULT_OUTPUT=json
```